munit-scalacheck/shared/src/main/scala/munit/scalacheck/ScalaCheckSuite.scala

```diff
@@ -0,0 +1,39 @@
+package munit
+package scalacheck
+
+import org.scalacheck.Prop
+import org.scalacheck.{Test => ScalaCheckTest}
+import org.scalacheck.util.Pretty
+import scala.concurrent.Future
+
+abstract class ScalaCheckSuite extends FunSuite {
+
+  def property(
+      options: TestOptions
+  )(body: => Prop)(implicit loc: Location): Unit = {
+    test(options)(body)
+  }
+
+  override def munitValueTransforms: List[ValueTransform] =
+    super.munitValueTransforms ++ List(
+      new ValueTransform("ScalaCheck Prop", {
+        case prop: Prop =>
+          val result = ScalaCheckTest.check(scalaCheckTestParameters, prop)
+          val prettyResult = Pretty.pretty(result, scalaCheckPrettyParameters)
+          if (result.passed) {
+            println(prettyResult)
```

Comment:
is there a smarter thing than `println` for reporting a successful test?

Reply:
This is OK. We have access to the sbt logger, which we could provide access to as an alternative. JUnit only supports a `didSucceed(description: Description): Unit` notification hook, a test success can't include more information.

Reply:
The IntelliJ integration BTW captures stdout per test case